### PR TITLE
Draft: Preview mode and Collections docs

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,6 +1,21 @@
 .Welcome!
 * xref:introduction:intro.adoc[Why Couchbase?]
 * xref:introduction:whats-new.adoc[What's New?]
+* xref:introduction:developer-preview/developer-preview-introduction.adoc[Developer Preview (Temp)]
```
Comment (simon-dew) [inline modules/ROOT/nav.adoc]: All of these ToC links point to **introduction:developer-preview/...**, but the new files created as part of this commit are all in a top-level **developer-preview** module.

File: modules/developer-preview/preview-mode.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
+= Developer Preview Mode and Features
```
Comment (simon-dew) [inline modules/developer-preview/preview-mode.adoc]: This looks like a duplicate of **developer-preview-introduction.adoc** but is not in the **pages** directory. Should this file be deleted?

File: modules/developer-preview/pages/preview-mode.adoc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+= Developer Preview Mode and Features
+
+[abstract]
```
Comment (simon-dew) [inline modules/developer-preview/pages/preview-mode.adoc:3]: This looks like a duplicate of **developer-preview-introduction.adoc** but has no content. Should this file be deleted?

**Review by simon-dew — CHANGES_REQUESTED** [review]
The developer preview introduction page will be really useful, thanks. Just a handful of things need to be addressed, I think. See comments in files.

File: modules/developer-preview/pages/preview-mode.adoc (hunk 3 above)
Comment (amarantha-k) [reply modules/developer-preview/pages/preview-mode.adoc:3]: This is the updated page that should stay. The original developer-preview-introduction.adoc   will be deleted.

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (amarantha-k) [reply modules/ROOT/nav.adoc]: Thanks.  nav.adoc has been updated now.

**Review by simon-dew — APPROVED** [review]

File: modules/developer-preview/pages/collections/manage-collections-with-cli.adoc [hunk 4]
```diff
@@ -0,0 +1,378 @@
+= Collections via CLI
+
+[abstract]
+_Collections_ can be managed by means of the Couchbase CLI.
+
+[#Using the CLI]
+== Using the CLI
+
+The following sequence of CLI commands demonstrates how to create, examine, and delete scopes and collections.
+It assumes that Couchbase Server has been installed, initialized, and provisioned on a _Ubuntu_ system.
```
Comment (simon-dew) [inline modules/developer-preview/pages/collections/manage-collections-with-cli.adoc:10]: I think it should be "an Ubuntu system", as the suggested pronunciation of Ubuntu is [ooboontoo](https://ubuntu.com/about). I'll raise a suggestion to add it to the style guide. Not a blocker.

File: modules/developer-preview/pages/collections/rest-api-reference/dropping-a-scope.adoc [hunk 5]
```diff
@@ -0,0 +1,61 @@
+= Dropping a Scope
+
+[abstract]
+Scopes can be _dropped_, by means of the REST API.
+
+== Description
+
+Scopes are dropped by means of the DELETE /pools/default/buckets/<bucket_name>/collections/<scope_name>  HTTP method and URI.
```
Comment (simon-dew) [inline modules/developer-preview/pages/collections/rest-api-reference/dropping-a-scope.adoc:8]: Consider making `<bucket_name>` and `<scope_name>` italic, as in **dropping-a-collection.adoc**.

File: modules/developer-preview/pages/preview-mode.adoc [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+For the Developer Preview, collections can be managed by either the _REST API_ or the _CLI_.
+For more information, see xref:developer-preview/collections/collections-overview.adoc[Collections Overview].
+
+* High Data Density
+
+* Cost-based Optimizer
+
+* Index Advisor
+
+* N1QL JavaScript UDFs (Stored Procedures)
```
Comment (simon-dew) [inline modules/developer-preview/pages/preview-mode.adoc:82]: Not just JavaScript UDFs, but inline UDFs also. Maybe rephrase as `N1QL JavaScript UDFs and Stored Procedures`.

File: modules/developer-preview/pages/preview-mode.adoc [hunk 7]
```diff
@@ -0,0 +1,87 @@
+= Developer Preview Mode and Features
+
+[abstract]
+The Developer Preview mode, also known as Preview mode, provides early access to upcoming features and enables you to play with these features and provide feedback during the development phase. Note that these features are not supported by Couchbase Legal Agreements, may not be functionally complete, and are not intended for production use. They are intended for development and testing purposes only. 
+
+Introduced in Couchbase Server Enterprise Edition 6.5 Beta, the Developer Preview mode is enabled by default during the Beta release. When the Preview mode is enabled, the cluster is converted to a Developer Preview cluster and cannot be switched back. Given that the Developer Preview features are under development, Couchbase cannot guarantee the stability of these features. As a result, you cannot upgrade to subsequent releases.
+
+While the Developer Preview Mode can be enabled using the CLI or REST API, once enabled, it cannot be disabled. 
+
+Note that the commands on this page assume a Ubuntu Linux environment. For the appropriate command-paths, see xref:cli:cli-intro.adoc[CLI Reference].
```
Comment (simon-dew) [inline modules/developer-preview/pages/preview-mode.adoc:10]: See previous comment. Would prefer "an Ubuntu" but be consistent with usage in the rest of the docs.

**Review by simon-dew — APPROVED** [review]
Second round of comments. No blockers.

**Comment by amarantha-k** [comment]
Local build successful, will merge and then fix the next set of comments in another PR.
